tests/core/test_model.py

```diff
@@ -2698,6 +2698,47 @@ def test_model_table_properties_conflicts() -> None:
         sql_model.physical_properties
 
 
+def test_session_properties_on_model_and_project(sushi_context):
```

Comment:
Let's also add a test when there are only defaults but nothing is set in the model definition.